src/main/js/App.js

```diff
@@ -6,20 +6,43 @@ import GlobalConfiguration from './component/GlobalConfiguration';
 import EmailConfiguration from './component/EmailConfiguration';
 import HipChatConfiguration from './component/HipChatConfiguration';
 import SlackConfiguration from './component/SlackConfiguration';
+import styles from '../css/main.css';
+
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
 
 class App extends React.Component {
-	render() {
-		return (
-				<div>
-				<GlobalConfiguration />
-				<EmailConfiguration />
-				<HipChatConfiguration />
-				<SlackConfiguration />
-				</div>
-		)
-	}
+    constructor() {
+        super();
+        this.state = {index: 0};
+    }
+
+    render() {
+        return (
+            <div>
+                <div className={styles.header}>
+                    <h1>Blackduck Software Alert</h1>
```

Comment:
This should be Black Duck Alert

Reply:
We don't want something like Blackduck Alert? I didn't think we wanted to separate the name Blackduck.